doc/source/commands/flight/systems.rst

```diff
@@ -25,7 +25,9 @@ RCS and SAS
 
     Turns the RCS **on** or **off**, like using ``R`` at the keyboard::
 
-        RCS ON.
+        RCS ON. // same as SET RCS TO TRUE.
+        RCS OFF. // same as SET RCS TO FALSE.
+        PRINT RCS.  // prints either "True" or "False".
```

Comment:
Technically not part of the remit of the issue, but I didn't like how terse the example was originally, and wanted to make clear the point that RCS is really just a boolean variable and can be used like one.